cli/src/commands/invite/greet.rs

```diff
@@ -61,6 +61,10 @@ pub async fn main(args: Args) -> anyhow::Result<()> {
             let ctx = step4_device(ctx).await?;
             step5_device(ctx).await
         }
+        InviteListItem::ShamirRecovery { .. } => {
+            // See: https://github.com/Scille/parsec-cloud/issues/8841
+            todo!("Implement shamir recovery greeting");
```

Comment:
Return a anyhow Error here, the error message should state the it's not currently supported

Reply:
f934021770e4c56615e8a46411c878ef962376a6